library/Kubernetes/Web/IngressListItem.php

```diff
@@ -8,38 +8,62 @@
 use Icinga\Module\Kubernetes\Common\Links;
 use ipl\Html\Attributes;
 use ipl\Html\BaseHtmlElement;
+use ipl\Html\HtmlDocument;
 use ipl\Html\HtmlElement;
+use ipl\Html\Text;
 use ipl\I18n\Translation;
+use ipl\Web\Widget\HorizontalKeyValue;
 use ipl\Web\Widget\Link;
 use ipl\Web\Widget\TimeAgo;
-use ipl\Web\Widget\VerticalKeyValue;
 
 class IngressListItem extends BaseListItem
 {
     use Translation;
 
     protected function assembleHeader(BaseHtmlElement $header): void
     {
-        $header
-            ->addHtml($this->createTitle())
-            ->addHtml(new TimeAgo($this->item->created->getTimestamp()));
+        $header->addHtml(
+            $this->createTitle(),
+            new TimeAgo($this->item->created->getTimestamp())
+        );
     }
 
     protected function assembleMain(BaseHtmlElement $main): void
     {
+        $main->addHtml(
+            $this->createHeader(),
+            $this->createFooter()
+        );
+    }
 
-        $main->addHtml($this->createHeader());
-
-        $keyValue = new HtmlElement('div', new Attributes(['class' => 'key-value']));
+    protected function assembleFooter(BaseHtmlElement $footer): void
+    {
+        $host = null;
```

Comment:
I think it is possible to set up multiple hosts, so it makes sense to create host as array and implode it with `, ` if it is not empty.